TOC.md

```diff
@@ -247,6 +247,18 @@
   + 系统数据库
     - [`mysql`](/reference/system-databases/mysql.md)
     - [`information_schema`](/reference/system-databases/information-schema.md)
+    + [`sql-diagnosis`](/reference/system-databases/sql-diagnosis.md)
```

Comment:
The PingCAP website does not support this kind of link (link of an item that has sub-items).